src/GRA.Web/Areas/MissionControl/Views/VendorCodes/Configure.cshtml

```diff
@@ -0,0 +1,213 @@
+﻿@model GRA.Domain.Model.VendorCodeType
+
+@if (Model.Id > 0)
+{
+    @await Html.PartialAsync("_VendorCodePartial")
+}
+
+<div class="row" style="margin-top: 1rem; margin-bottom: 1rem;">
+    <div class="col-xs-12">
+        <p>Vendor codes are not yet configured. Use this page to set them up.</p>
+        <ul>
+            <li>
+                If you wish to configure a vendor code to be awarded with a trigger, you need only
+                configure the top fields.
```

Comment:
I would change the wording on the last part of this sentence:
"you need only configure the top fields."